Source/WebGPU/WGSL/Overload.cpp

```diff
@@ -216,7 +225,7 @@ std::optional<ViableOverload> OverloadResolver::considerCandidate(const Overload
             if (!first)
                 dataLog(", ");
             first = false;
-            dataLog(*materialize(parameter));
+            dataLog(**materialize(parameter));
```

Comment:
Optional pointer? That doesn't smell right. Pointers can already be optional 🤔

Reply:
Oops, good call, not sure why I did that 🫣